channels/alpha

```diff
@@ -20,8 +20,10 @@ spec:
     requiredVersion: 1.4.2
   kopsVersions:
   - range: ">=1.5.0-alpha1"
-    recommendedVersion: 1.5.0-beta1
-    #requiredVersion: 1.5.0-beta1
+    recommendedVersion: 1.5.0-beta2
```

Comment:
Future consideration..

Do we want to putting alpha releases in the `alpha` channel? Seems a bit counter intuitive.. maybe a better name?